docs/latex/Inhalt/03_Inhalt/01_einleitung.tex

```diff
@@ -15,4 +15,4 @@ \section{Funktionalität}
 Als Benutzeroberfläche für die Software ist eine \ac{CLI} vorgesehen.
 
 \section{Technologien}
-Für die Umsetzung wird Java als Programmiersprache und Laufzeitumgebung verwendet. Für die Paketverwaltung wird Maven, für das Parsen des HTML Jsoup und zur Kommunikation mit der Datenbank werden JPA und Hibernate verwendet. Als Datenbankmanagementsystem wird H2 verwendet. Das Projekt befindet sich durch Git bzw. GitHub unter Versionsverwaltung.
+Für die Umsetzung wird Java als Programmiersprache und Laufzeitumgebung verwendet. Für die Paketverwaltung wird Maven, für das Parsen des HTML Jsoup und zur Kommunikation mit der Datenbank werden JPA und Hibernate verwendet. Als Datenbankmanagementsystem wird H2 verwendet. Das Projekt befindet sich durch Git bzw. GitHub unter Versionsverwaltung. Der Source Code ist im Repository \href{https://youtu.be/dQw4w9WgXcQ}{https://github.com/anditru/quickie} öffentlich einsehbar.
```

Comment:
Noch anmerken, welchen commit-Stand wir in allen Kapiteln außer "Refactoring" beschreiben

Reply:
Sehr schöner Link